src/ast/mod.rs

```diff
@@ -878,6 +880,16 @@ where
                 expr,
                 schedule,
             } => list!("simplify", schedule, expr),
+            GenericCommand::Datatypes { span: _, datatypes } => {
+                let datatypes: Vec<_> = datatypes
+                    .iter()
+                    .map(|(_, name, variants)| match variants {
+                        Ok(variants) => list!(name, ++ variants),
+                        Err((head, args)) => list!("sort", name, list!(head, ++ args)),
+                    })
+                    .collect();
+                list!("datatypes", ++ datatypes)
```

Comment:
Should this be `datatype*`? Hm, I wonder if there are no tests for these?

Reply:
Good catch! For `NCommand`, cases like this would be caught by resugared tests (which re-run same tests over programs converted from ASTs), but we don't have tests for Command (which is immediately desugared into `NCommand`)